docs/FAQ.md

```diff
@@ -0,0 +1,98 @@
+
+### CUDA (Default GPU) vs CPU?
+The CPU version of ONNX Runtime provides a complete implementation of all operators in the ONNX spec. This ensures that your ONNX-compliant model can execute successfully (*minor caveat: not 100% of types are supported to minimize the binary size, but can usually be easily added as needed*). On the other hand, not all CUDA kernels are implemented, as these have been prioritized on an as-needed basis. As a result, this means that if your model contains operators that do not have a CUDA implementation, it will fall back to CPU. Switching between CPU and GPU can cause significant performance impact.
+
+If you require a specific operator that is not currently supported, please consider [contributing](./../CONTRIBUTING.md) and/or [file an issue](https://github.com/microsoft/onnxruntime/issues) clearly describing your use case and share your model if possible. 
+
+### TensorRT vs CUDA?
+TensorRT and CUDA are separate execution providers for ONNX Runtime. On the same hardware, TensorRT will generally provide better performance; however, this depends on the specific model and whether the operators in the model can be supported by TensorRT. In cases where TensorRT cannot handle the subgraph(s), it will fall back to CUDA. Note that the TensorRT EP may depend on a different version of CUDA than the CUDA EP. 
+
+### TensorRT/CUDA vs DirectML? 
+DirectML is the hardware-accelerated DirectX 12 library for machine learning on Windows and supports all DirectX 12 capable devices (Nvidia, Intel, AMD). This means that if you are targeting Windows GPUs, using the DirectML Execution Provider is likely your best bet. This can be used with both the ONNX Runtime as well as [WinML APIs](./WinRT_API.md).
+
+## Do the GPU builds support quantized models?
+The default CUDA build does not support any quantized operators right now. The TensorRT EP has limited support for INT8 quantized ops. In general, support of quantized models through ORT is continuing to expand on a model-driven basis. For performance improvements, quantization is not always required, and we suggest trying alternative strategies to [performance tune](./ONNX_Runtime_Perf_Tuning.md) before determining that quantization is necessary.
+
+## Does the Python API add any overhead to performance?
+The Python API doesn't add any overhead to performance when using the CPU execution provider only. When using the GPU provider, inputs and outputs need to be copied from CPU to GPU and vice-versa and the Python API doesn't allow inputs/outputs to be setup for this to be done prior to execution (calling run). Hence this copy needs to be done on the fly as part of execution. This eats into the execution time thereby impacting performance.
+
+## How do I turn on verbose mode logging in Python?
```

Comment:
I think we should expand this to the C API as well and modify the question as follows:
Q: How do I change the severity level of the default logger to something other than the default (WARNING)?
A: (1) For Python, use the following snippet of code. [Here](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/onnxruntime_pybind_state.cc#L367) are valid values for the severity levels.  
(2) For C, use the following API: SetSessionLogSeverityLevel. [Here](https://github.com/microsoft/onnxruntime/blob/master/include/onnxruntime/core/session/onnxruntime_c_api.h#L105-L111) are the valid values for the severity levels.

Setting the severity level to VERBOSE is most useful when debugging errors.